dev-tools/omicron-dev/src/bin/omicron-dev.rs

```diff
@@ -320,6 +332,116 @@ async fn cmd_clickhouse_run(args: &ChRunArgs) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+async fn start_replicated_cluster() -> Result<(), anyhow::Error> {
+    // Start a stream listening for SIGINT
+    let signals = Signals::new(&[SIGINT]).expect("failed to wait for SIGINT");
+    let mut signal_stream = signals.fuse();
+
+    // Start the database server and keeper processes
+    let cur_dir = std::env::current_dir().unwrap();
+    let replica_config =
+        cur_dir.as_path().join("oximeter/db/src/configs/replica_config.xml");
+    let cur_dir = std::env::current_dir().unwrap();
+    let keeper_config =
+        cur_dir.as_path().join("oximeter/db/src/configs/keeper_config.xml");
+
+    let mut cluster =
+        dev::clickhouse::ClickHouseCluster::new(replica_config, keeper_config)
+            .await?;
+    println!(
+        "omicron-dev: running ClickHouse cluster with configuration files:\n \
+        replicas: {}\n keepers: {}",
+        cluster.replica_config_path().display(),
+        cluster.keeper_config_path().display()
+    );
+    let pid_error_msg = "Failed to get process PID, it may not have started";
+    println!(
+        "omicron-dev: ClickHouse cluster is running with PIDs: {}, {}, {}, {}, {}",
```

Comment:
Could we print out which kind of process each corresponds to? E.g., `keeper PIDS = [x, y, z]`.